gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -736,15 +754,20 @@ gst_tensor_filter_parse_accelerator (GstTensorFilterPrivate * priv,
     }
   }
 
-  /** Convert the list to string format */
-  accl_support = g_malloc (sizeof (gchar *) * (info->num_hw + 1));
+  /**
+   * Convert the list to string format
+   * Extra 2 entries for basic accelerators : auto and default
+   */
+  accl_support = g_malloc (sizeof (gchar *) * (info->num_hw + 2 + 1));
   if (!accl_support)
```

Comment:
This error handing will be never called. Let's remove it.

Reply:
done.